xbmc/utils/StringUtils.h

```diff
@@ -134,9 +134,87 @@ class StringUtils
    \param delimiter Delimiter to be used to split the input string
    \param iMaxStrings (optional) Maximum number of splitted strings
    */
-  static std::vector<std::string> Split(const std::string& input, const std::string& delimiter, unsigned int iMaxStrings = 0);
-  static std::vector<std::string> Split(const std::string& input, const char delimiter, size_t iMaxStrings = 0);
-  static std::vector<std::string> Split(const std::string& input, const std::vector<std::string> &delimiters);
+  static std::vector<std::string> Split(const std::string& input, const std::string& delimiter, unsigned int iMaxStrings = 0)
```

Comment:
I can put Split() back in the .cpp, but SplitTo() would have to stay because it is a template.
I wanted to use a std::set for the EGL extensions because it is more appropriate than a std::vector. But Split() only returned a std::vector which I would then have to copy to a std::set, which seemed pointless to me.
If you want to avoid the StringUtils change I can revert it and copy the vector to the set after splitting.